{{cookiecutter.project_slug}}/clients/react/src/utils/auth.tsx

```diff
@@ -12,38 +12,53 @@ import { PUBLIC_ROUTES } from "./routes";
 // 2a. unset localStorage token & exp on logout
 // 2b. call client.clearStore() on logout
 
-export const AuthContext = React.createContext({
+const AUTH_TOKEN_KEY = 'auth-token'
```

Comment:
I see. Interesting take. If the apps that use this module are different, they would not be using the same storage, right 🤔?  unless you're in development and visited localhost in a non-incognito window after switching apps. 

Maybe there are some other edge cases where that could happen so I like the idea since it attempts to prevent that potential key collision.

So surely an environment variable could do the job. Indeed there's an npm env variable that we could use that would take advantage of the cookie cutter templating in package.json:

```
REACT_APP_NAME=$npm_package_name
```
This would take package.json's `name` value